src/core/qtlibtorrent/qbtsession.cpp

```diff
@@ -575,6 +575,8 @@ void QBtSession::configureSession() {
     }else{
         disableIPFilter();
     }
+    sessionSettings.apply_ip_filter_to_trackers = pref->isFilteringTrackerEnabled();
+    setSessionSettings(sessionSettings);
```

Comment:
Yes, put line 578 somewhere above line 519. Try to find even a remotely related setting to group it together. If not, then put it in the begining.
Delete line 579.